pyproject.toml

```diff
@@ -53,13 +53,14 @@ s3fs = { version = ">= 2023.4.0", optional = true }
 adlfs = { version = ">= 2023.4.0", optional = true }
 kfp = { version = "2.0.1", optional = true }
 pandas = { version = ">= 1.3.5", optional = true }
+google-cloud-aiplatform = "^1.32.0"
```

Comment:
Can you add it as optional similar to the above?

Reply:
good spot, updated!